### PR TITLE
Milstone5 :Styling

### PR DESCRIPTION
**Pull request by Favourezeugwa**
## In this milestone, 
- I Used the information detailed in Zeplin to style the website.

### CODE REVIEW

File: src/Components/Navigation.js [hunk 1]
```diff
@@ -18,6 +20,7 @@ const Navigation = () => (
         </Link>
       </li>
     </ul>
+    <FaUser className="user-icon" style={{ color: '#0290ff', cursor: 'pointer' }} />
```
Comment (Ismailco) [inline src/Components/Navigation.js]: - [ ] Inline styles are not allowed you can use the React inline styles by adding the styles into a variable and passing it into the component/tag, kindly implement this so your code follows the requirements
e.g:
```
const myComponentStyle = {
  color: 'blue',
  lineHeight: 10,
  padding: '1.5em',
}

function MyComponent(){

    return <div style={myComponentStyle}> Inline Styled Component</div>

}
```

File: src/App.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import Categories from './Components/Categories';
 
 function App() {
   return (
```
Comment (Ismailco) [inline src/App.js:10]: - [ ] Your app should be identical to the provided styling, kindly remove the outer space with the background so the app can take the whole space

|Your design|How it should be|
|-|-|
|Screenshot_20220716_171436|Screenshot_20220716_171504|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Favourezeugwa, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keneogu — APPROVED** [review]
Hi @Favourezeugwa,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**